Code/GraphMol/SynthonSpaceSearch/substructure_search_catch_tests.cpp

```diff
@@ -559,4 +559,19 @@ TEST_CASE("DOS File") {
   SynthonSpace synthonspace;
   synthonspace.readTextFile(libName);
   CHECK(synthonspace.getNumProducts() == 12);
+}
+
+TEST_CASE("Synthon Error") {
+  REQUIRE(rdbase);
+  std::string fName(rdbase);
+  {
+  std::string libName = fName + "/Code/GraphMol/SynthonSpaceSearch/data/amide_space_error.txt";
+  SynthonSpace synthonspace;
+  CHECK_THROWS(synthonspace.readTextFile(libName));
```

Comment:
it wouldn't be terrible if the formatting here was consistent

Reply:
Yep.  Tidy code is important :-).